app/services/csv/sales_log_csv_service.rb

```diff
@@ -0,0 +1,108 @@
+module Csv
+  class SalesLogCsvService
+    def initialize(export_type:)
+      @export_type = export_type
+      @attributes = sales_log_attributes
+    end
+
+    def prepare_csv(logs)
+      CSV.generate(headers: true) do |csv|
+        csv << @attributes
+
+        logs.find_each do |log|
+          csv << @attributes.map { |attribute| value(attribute, log) }
+        end
+      end
+    end
+
+  private
+
+    ATTRIBUTES_OF_RELATED_OBJECTS = {
```

Comment:
```
irb(main):001:0> class A
irb(main):002:1>   private
irb(main):003:1>   B = "B"
irb(main):004:1> end
=> "B"
irb(main):005:0> A::B
=> "B"
```
As an fyi declaring constants after a `private` this way does not make them private